components/Chat/ClientChatPage.tsx

```
@@ -0,0 +1,361 @@
+              // Check if we need to update the URL (if it doesn't already contain the chat ID)
+              if (!currentPath.endsWith(`/${chatId}`)) {
+                const newPath = `/${encodedName}/chat/${chatId}`;
+                console.log(`Updating URL to: ${newPath}`);
+                router.replace(newPath, { scroll: false });
+              }
+            }
+          }
+        }
+      },
+      onError: (error: Error) => {
+        toast.error("Error: " + (error.message || "Failed to send message"));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)